erizo/src/erizo/media/ExternalOutput.cpp

```diff
@@ -390,9 +392,29 @@ void ExternalOutput::queueData(char* buffer, int length, packetType type){
             ELOG_DEBUG("File %s, video offset msec: %llu", context_->filename, videoOffsetMsec_);
         }
 
+        RtpHeader* h = reinterpret_cast<RtpHeader*>(buffer);
+
+
+
+        if (firstVideoTimestamp_ != -1){ //If not first frame
+            //Calculate video timestamp
+            long long currentTimestamp = h->getTimestamp();
+            if (currentTimestamp - firstVideoTimestamp_ < 0) {
+                // we wrapped.  add 2^32 to correct this.  We only handle a single wrap around since that's ~13 hours of recording, minimum.
+                currentTimestamp += 0xFFFFFFFF;
+            }
+
+            long long videoTimeStamp = (currentTimestamp - firstVideoTimestamp_) / (90000 / video_stream_->time_base.den);
```

Comment:
I figured out where the 90000 comes from.  This is because all of the video streams used in licode use a 90khz clock.